src/main/msp/msp.c

```diff
@@ -1418,6 +1418,8 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, 0);
         sbufWriteU16(dst, 0);
 #endif
+        // Added in MSP API 1.41
+        sbufWriteU8(dst, currentPidProfile->dterm_filter2_type);
```

Comment:
I think is better to include it before the DYN_FILTER, line 2060, to be with the other similar elements, with one byte more in the IF. The dyn_filter has been added recently and all of them are from the same version. What do you think?

Reply:
Yes, good point. Also reinforces my point that we need to become more consequent in adding comments about what version something was added in - this way being able to move stuff around inside a version doesn't require external information.

Fixed.